pkg/rancher-desktop/components/Preferences/BodyVirtualMachine.vue

```diff
@@ -59,12 +59,18 @@ export default Vue.extend({
         },
       );
     },
+    selectTab(tab: string) {
+      const tabbed = this.$refs.tabbed as RdTabbed;
+
+      tabbed.$children[0].select(tab);
+    },
```

Comment:
We shouldn't need to reach into `$children` to invoke `select` and should adhere to the pattern of "Props Down, Events Up". What I mean by this is that instead of each child component adding a `selectTab` function prop to pass down the component tree, is that child components should instead emit an event with the value that contains the tab name that we want to navigate to.